src/Game/zMusic.h

```diff
@@ -1,6 +1,36 @@
 #ifndef ZMUSIC_H
 #define ZMUSIC_H
 
+#include <types.h>
 
+struct zVolumeInfo {
+	float32 cur;
+	float32 end;
+	float32 inc;
+	uint32 adjusted[2];
+};
+
+struct zMusicTrackInfo {
+	uint32 snd_id;
+	uint32 loop;
+	int32 unk;
+	uint32 assetID;
+	float32 lastVol;
+};
+
+enum LevelMusic {
+	//TODO: POPULATE MUSIC LISTING!
+	Null
+};
```

Comment:
It appears that this enum doesn't exist in the dwarf data and that this was made up. Avoid making up enums or types that don't exist anywhere in the mangled names or dwarf data. We want to keep the codebase strictly to how they typed things for now. Documentation or cleaning up things like this will be done at some point far in the future (if at all).

You're using your custom enum here, in a commented out function:
`LevelMusic getCurrLevelMusicEnum()`

The type of this function in the dwarf data is an `int32`: 
`int32 getCurrLevelMusicEnum();`

Was it an enum in the original source? Probably. Maybe there is an enum for this somewhere in the dwarf data, but if you can't find it, just leave it as an int. This also means you shouldn't be casting every single int into your new enum type.